element_calcium_imaging/imaging_no_curation.py

```diff
@@ -517,7 +517,7 @@ def make(self, key):
 
                         plane_processing_tasks = []
                         for plane_idx in PVmeta.meta["plane_indices"]:
-                            pln_output_dir = output_dir / f"pln{plane_idx}_chn{channel}"
+                            pln_output_dir = pathlib.Path(output_dir + f"/pln{plane_idx}_chn{channel}")
```

Comment:
Why do we need this?
`output_dir` should already be a `Path` object at this step in the code

Reply:
I get this error if I use the `/` there: `TypeError: unsupported operand type(s) for /: 'str' and 'str'`